tools/nni_cmd/config_schema.py

```diff
@@ -262,6 +262,30 @@ def validate(self, data):
     }
 }
 
+adl_trial_schema = {
+    'trial':{
+        'codeDir': setType('codeDir', str),
+        'command': setType('command', str),
+        'gpuNum': setNumberRange('gpuNum', int, 0, 99999),
+        'image': setType('image', str),
+        Optional('imagePullSecrets'): [{
+            'name': setType('name', str)
+        }],
+        Optional('nfs'): {
```

Comment:
Generally, this kind of storage configuration is put into `adlConfig`, for example `kubeflowConfig` and `frameworkControllerConfig`

Reply:
Tiendang130196@gmail.com

Reply:
@NiuChan1301 - please remove your invalid review comments to avoid confusion. 

@pw2393 - please ignore @NiuChan1301's review comments if they did not provide any valid suggestions, (s)he is not a required reviewer.